scripts/settings.js

```diff
@@ -44,8 +44,11 @@ export let cExplorer = cChainParams.current.Explorers[0];
 export let cNode = cChainParams.current.Nodes[0];
 /** A mode which allows MPW to automatically select it's data sources */
 export let fAutoSwitch = true;
-/** The active Cold Staking address: default is the PIVX Labs address */
-export let strColdStakingAddress = 'SdgQDpS8jDRJDX8yK8m9KnTMarsE84zdsy';
+/** The active Cold Staking address: default is set via chainparams */
+export let strColdStakingAddress =
+    cChainParams.current.defaultColdStakingAddress;
+/** The primary Owner Address for Cold Staking: default is a new address each delegation */
+export let strColdStakeOwnerAddress = '';
```

Comment:
why are `strColdStakeOwnerAddress` and `strColdStakingAddress` in settings?

Reply:
They are customisable (permanently) by users.